src/test/java/jenkins/plugins/okhttp/api/ClientTest.java

```diff
@@ -63,12 +62,14 @@ private void secureProxy(String authenticationType, final String... authorizedUs
                         .withStatus(407) // 407: Proxy Authentication Required
                         .withHeader("Proxy-Authenticate", authenticationType)));
 
-        if (authorizedUserPass != null && "Basic".equals(authenticationType)) {
-            for (String userPass : authorizedUserPass) {
-                final String authenticationHeaderValue = "Basic " + Base64.getEncoder().encodeToString(userPass.getBytes(StandardCharsets.UTF_8));
-                proxy.stubFor(get(WireMock.anyUrl())
-                        .withHeader("Proxy-Authorization", WireMock.equalTo(authenticationHeaderValue))
-                        .willReturn(WireMock.ok("Hello from proxy")));
+        if (authorizedUserPass != null && authenticationType != null) {
+            if (authenticationType.toLowerCase().contains("basic") || authenticationType.equalsIgnoreCase("okhttp-preemptive")) {
```

Comment:
```suggestion
            if (authenticationType.toLowerCase(Locale.ROOT).startsWith("basic")) {
```
see also discussion on the test.  this header should not be seen on the wire IIUC.

Reply:
Done.